libraries/mbed/targets/hal/TARGET_NXP/TARGET_LPC176X/TARGET_XBED_LPC1768/PinNames.h

```diff
@@ -82,7 +83,8 @@ typedef enum {
 	SCL = P0_28,
 	ISP = P2_10,
 	CLK = P1_27,
```

Comment:
This may be caused by the editor of our partner. They use "tab" instead of 4 spaces.